### PR TITLE
feature: binance smart chain sett support

### PR DESCRIPTION
**Pull request by axejintao**

### CODE REVIEW

File: src/protocols/common/swap-util.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const ethPrice = (t0Price.eth * pair.reserve0 + t1Price.eth * pair.reserve1) / pair.totalSupply;
+  return {
+    address: contract,
+    usd: usdPrice,
+    eth: ethPrice,
+  };
+};
+
+export const getSushiswapPrice = async (contract: string): Promise<TokenPrice> => {
+  return getLiquidityPrice(SUSHISWAP_URL, contract.toLowerCase());
```
Comment (axejintao) [inline src/protocols/common/swap-util.ts:36]: @lookfirst is this what you had in mind?

Comment (lookfirst) [reply src/protocols/common/swap-util.ts:36]: yes, except you can put the toLowerCase within getLiqudityPrice()...

Comment (axejintao) [reply src/protocols/common/swap-util.ts:36]: wasn't that how it was before?

Comment (axejintao) [reply src/protocols/common/swap-util.ts:36]: nvm, i misunderstood the original comment - makes sense. just to lower the occurrences and funnel to one place

Comment (lookfirst) [reply src/protocols/common/swap-util.ts]: yep... funnel the logic such that you only call toLowerCase() once...

**Review by ethkaneki — APPROVED** [review]

**Pull request merged**